src/http/get-home-000view/connected-ticket.js

```diff
@@ -0,0 +1,40 @@
+const data = require('@begin/data')
+const { Connection } = require('./connection')
+const { Ticket } = require('./ticket')
+
+class ConnectedTicket {
+  constructor(ticket) {
+    this._ticket = ticket
+  }
+
+  async connections() {
+    const connections = await Connection.connectionsForTicketKey(this._ticket.key)
+    const joinedToConnections = connections.map((connection) => this._joinToConnection(connection))
+    return Promise.all(joinedToConnections)
+  }
+
+  async addConnection(toTicket) {
+    const canConnect = await Connection.canConnect(this._ticket, toTicket)
+    if (!canConnect) {
+      await this._incrementBadConnection()
+      return
+    }
+    await Connection.addConnection(this._ticket, toTicket)
+  }
+
+  async _incrementBadConnection() {
```

Comment:
I wanted to use private properties and methods, but the `ecmaVersion: 2020` set in the eslint does not allow for it.